File: WebDriver-Series/WebDriver.Series.Tests/Controls/GridFilter.cs
using WebDriver.Series.Tests.Enums;

namespace WebDriver.Series.Tests.Controls
{
    public class GridFilter
    {
        public GridFilter(string columnName, FilterOperator filterOperator, string filterValue)
        {
            this.ColumnName = columnName;
            this.FilterOperator = filterOperator;
            this.FilterValue = filterValue;
        }

        public string ColumnName { get; set; }

        public FilterOperator FilterOperator { get; set; }

        public string FilterValue { get; set; }
    }
}

File: java/WebAutomation-Series/src/test/java/kendogrid/components/GridFilter.java
package kendogrid.components;

public class GridFilter {
    private String columnName;
    private FilterOperator filterOperator;
    private String filterValue;

    public GridFilter(String columnName, FilterOperator filterOperator, String filterValue) {
        this.columnName = columnName;
        this.filterOperator = filterOperator;
        this.filterValue = filterValue;
    }

    public String getColumnName() {
        return columnName;
    }

    public void setColumnName(String columnName) {
        this.columnName = columnName;
    }

    public FilterOperator getFilterOperator() {
        return filterOperator;
    }

    public void setFilterOperator(FilterOperator filterOperator) {
        this.filterOperator = filterOperator;
    }

    public String getFilterValue() {
        return filterValue;
    }

    public void setFilterValue(String filterValue) {
        this.filterValue = filterValue;
    }
}
